playground/package.json

```diff
@@ -4,9 +4,9 @@
   "description": "A starter example for building a Merchant Center application",
   "private": true,
   "scripts": {
-    "build": "mc-scripts build",
+    "build:playground": "mc-scripts build",
```

Comment:
Hmm why did you rename this? 🤔

Reply:
I see...well, on CI we build the playground anyway. So maybe it's not a bad idea to build it together with the other packages?

Reply:
PS: you can always "exclude" some packages from running lerna commands

Reply:
So that would be `lerna run build --ignore "merchant-center-starter-app"`

Reply:
I would keep `build` for building the package. Building an "example" like playground is better suited as its own command imho.

Reply:
I would like to keep `build`, because this is meant to be also an example on how to setup a project. I provided a couple of options on how to deal with this:
- ignore the playground package from the `lerna run build` command
- let it also build this package

Reply:
Ah, sorry I got confused. I thought `playground` is nested inside `application-shell`! I'm on board then :)

Reply:
Done in [637015263b0534fe1d3d047008032cf93574e67e](https://github.com/commercetools/merchant-center-application-kit/pull/40/commits/637015263b0534fe1d3d047008032cf93574e67e)